sae_lens/toolkit/pretrained_sae_loaders.py

```diff
@@ -451,32 +449,50 @@ def get_dictionary_learning_config_1(config: dict[str, Any]) -> dict[str, Any]:
     }
 
 
+def get_conversion_loader_name(sae_info: Optional[PretrainedSAELookup]):
+    conversion_loader_name = "sae_lens"
+    if sae_info is not None and sae_info.conversion_func is not None:
+        conversion_loader_name = sae_info.conversion_func
+    if conversion_loader_name not in NAMED_PRETRAINED_SAE_LOADERS:
+        raise ValueError(
+            f"Conversion func '{conversion_loader_name}' not found in NAMED_PRETRAINED_SAE_LOADERS."
+        )
+    return conversion_loader_name
+
+
+def get_sae_config(
+    release: str, sae_id: str, options: SAEConfigLoadOptions
+) -> dict[str, Any]:
```

Comment:
nit: this function should be covered by unit tests. benchmark tests don't get run in CI so don't really add any guarantees to the codebase. It should be cheap to load configs in a unit test since we don't need to download the whole model weights now. This could look like trying to load a specific config for each of the types of SAEs we have loaders for, and just asserting that the resulting config dict matches what we expect.

Reply:
I've added tests for each item in `NAMED_PRETRAINED_SAE_LOADERS`. Have we thought about splitting the tests in the `unit` directory into unit tests and integration tests?

The tests I've added are technically integration tests since they're interacting with the filesystem and downloading from Hugging Face. Though I think it's more important for the tests to be useful than to fit the definition of "unit tests".

Reply:
Definitely agree the word "unit" is not technically correct and is probably confusing, but I also don't think proper unit vs integration split matters here as the codebase isn't that large and we don't want to, e.g., only run the unit and not integration tests to save time. The majority of the tests in "test/unit" are technically integration tests since we don't mock out dependencies unless we absolutely have to and will happily run against real (tiny) models / data in tests.

IMO we just need to make sure functionality is covered by tests, whether unit or integration or both, and we shouldn't worry much about the distinction. I would normally just say "tests"  instead of "unit tests", but since we have a folder "tests/unit" which is run by CI and "tests/benchmark" which isn't, want to emphasize the "test/unit" folder. I'm not sure what to call the non-benchmark tests that isn't confusing, but also want as much as possible for people to add to tests that are run by CI and not add tests to "test/benchmark" which are never run.

Maybe we can move "tests/benchmark" into just "benchmark", and then move "tests/unit" into "tests", and then delete the "tests/acceptance" folder which is empty anyway? Probably worth discussing in an issue.